benchmarks/lib/src/jmh/java/com/amazon/corretto/crypto/provider/benchmarks/SignatureRSA.java

```diff
@@ -0,0 +1,42 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package com.amazon.corretto.crypto.provider.benchmarks;
+
+import java.security.spec.RSAKeyGenParameterSpec;
+
+import com.amazon.corretto.crypto.provider.AmazonCorrettoCryptoProvider;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+
+@State(Scope.Benchmark)
+public class SignatureRSA extends SignatureBase {
+  @Param({"SHA1", "SHA224", "SHA256", "SHA384", "SHA512"})
```

Comment:
As per the note `SHA256withRSA <- Fix the hash to SHA256`, we should only benchmark this algorithm for SHA256. It's not particularly interesting to us to parameterize across all the different hash algorithms since those are already benchmarked elsewhere.

Reply:
+1. this will also prevent the visualization tool from running out of unique colors.